raster_loader/io/bigquery.py

```diff
@@ -0,0 +1,259 @@
+import time
+import json
+import pandas as pd
+import rasterio
+
+from raster_loader.errors import import_error_bigquery, IncompatibleRasterException
+from raster_loader.utils import ask_yes_no_question, batched
+from raster_loader.io.common import (
+    rasterio_metadata,
+    rasterio_windows_to_records,
+    get_number_of_blocks,
+    check_metadata_is_compatible,
+    update_metadata,
+)
+
+from typing import Iterable, List, Tuple
+from functools import partial
+
+try:
+    from google.cloud import bigquery
+except ImportError:  # pragma: no cover
+    _has_bigquery = False
+else:
+    _has_bigquery = True
+
+from raster_loader.io.datawarehouse import DataWarehouse
+
+
+class BigQuery(DataWarehouse):
```

Comment:
I would rename it as `BigQueryConnection`

Reply:
Updated